pydis_site/README.md

```diff
@@ -0,0 +1,63 @@
+# `pydis_site` project directory
+
+This directory hosts the root of our **Django project**[^1], and is responsible
+for all logic powering our website. Let's go over the directories in detail:
+
+- [`apps`](./apps) contains our **Django apps**, which are the building blocks
+  that make up our Django project. A Django project must always consist of one
+  or more apps, and these apps can be made completely modular and reusable
+  across any Django project. In our project, each app controls a distinct part
+  of our website, such as the API or our resources system.  For more
+  information on reusable apps, see the official Django tutorial, [which has a
+  section on reusable
+  apps](https://docs.djangoproject.com/en/dev/intro/reusable-apps/). To learn
+  more about our specific apps, see the README inside the app folder itself.
```

Comment:
This part could really use a newline or two. It's a bit dense.

Reply:
hmm, wasn't aware markdown supported that. Let's do it.

Reply:
[8eca5c8](https://github.com/python-discord/site/pull/683/commits/8eca5c8ddeed9ef7b827da9f6e87df034c073d6d)